modules/luci-base/luasrc/sys.lua

```diff
@@ -358,6 +358,32 @@ function net.conntrack(callback)
 					if entry[key] == nil then
 						entry[key] = val
 					end
+					if entry["ptype"] == nil then
```

Comment:
This list is arbitrary and incomplete. I think it will cause confusion in the long run... ("Why is protocol X not show?")

Reply:
@jow- so drop the entire port type part right?

Reply:
Yes. If we want to introduce something like this it should be some kind of global configuration file similar to /etc/protocols

Reply:
yes the first idea was to use the /etc/protocols file but also that file is incomplete... will take a look about that.